### PR TITLE
Simplify time reference

### PR DESCRIPTION
**Pull request by chaals**
If you don't provide a reference that is relative to "now" you don't have to keep updating it.

We could use the date the web went live instead. I don't think this paragraph is critical in any case.

### CODE REVIEW

File: Vision/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## Introduction
-The World Wide Web was conceived more than 25 years ago 
+The World Wide Web was originally conceived at the end of the 1980s
```
Comment (cwilso) [inline Vision/README.md:2]: I would point out that the original framing won't be untrue five years from now, or ten years from now, or fifteen years from now, so I'm not sure we need to fix this; but this somehow reads as more complex and less readable to me.  I'd prefer to simply drop the "more than 25 years ago" and insert the word "originally" as you did, removing the time reference altogether, if we want to make a change here.

**Review by tantek — CHANGES_REQUESTED** [review]
This feels like fixing something that's not broken so I'm not sure this is worth the effort.

One point of context that seems to be lost is that The Vision was written with an eye towards "the next 25 years of the web", and thus this reference to "more that 25 years ago" was meant to convey a parallel passage of a similar chunk of time, a past era as it were.

I'm not sure if that context is still relevant or merely a vestige of the development of this document. I'd like to hear @dwsinger's opinion as I believe it was he who proposed the "next 25 years" framing.

Otherwise I'm ok closing without making any changes.

**Comment by dwsinger** [comment]
I'd be fine with "originally conceived" and not even bother with the references to the 80s. You're right, dating isn't needed.

**Comment by cwilso** [comment]
Rolled in to #55.

**Pull request closed without merging**